### ISSUE TITLE
A few bugs.

### ISSUE THREAD

**Issue by spencerwooo** [felixse/FluentTerminal#144]
![image](https://user-images.githubusercontent.com/32114380/49272710-843e6000-f4ad-11e8-8c78-239c62b4b186.png)

First of all, I must say: **Fluent Terminal is the future for Windows console!** (For me)
It's just what I pictured - modern and fluent. I recommend it over Hyper, after using it a bit. So I look forward to seeing it on Microsoft Store. 

There are some bugs (I'm running Fluent Terminal on Windows 1809, with WSL environments):

- `clear` only works after I run it twice, once makes the whole panel's text disappear. (Triggers randomly)

![2018-11-30_15-09-01](https://user-images.githubusercontent.com/32114380/49274057-f44ee500-f4b1-11e8-8b2d-79067ca2a991.gif)

- Adjusting the opacity crashes the window. (Triggers randomly)
- `variable-width characters` rendering is still an issue, which appears to be an issue with `xterm.js`. I'm also having this issue with Hyper.
- Emoji, emoji, emoji. When can any terminal on Windows have proper Emoji support. ...( ＿ ＿)ノ｜

**Comment by spencerwooo** [comment]
A follow-up issue:

![image](https://user-images.githubusercontent.com/32114380/49275321-edc26c80-f4b5-11e8-9089-61c6673cfda6.png)

When running a dev server locally, I expect to `Ctrl + Click` on the url and invoke my default browser. Fluent Terminal didn't implement this. Hope you guys can look into this.

**Comment by skipmeister123** [comment]
I don't have that issue with running `clear`.  Maybe it's something with your environment?
![fluent terminal 2018-11-30 15-05-40_trim](https://user-images.githubusercontent.com/1911919/49313414-2ca3f280-f4b5-11e8-86d4-cb068f0d98d9.gif)
A lot of us use [Nerd Fonts](http://nerdfonts.com/) for emoji.  Maybe give that a shot and see if it solves that problem?

**Comment by spencerwooo** [comment]
Thank you! I'll definitely try Nerd Font!

`clear` still doesn't work...sometimes. I don't know what triggers it, but it fails occasionally. I'll try reinstalling Fluent Terminal and see if that solves the problem.

**Comment by spencerwooo** [comment]
**Follow up: Nope, emoji still doesn't work.** 

![2018-12-01_13-38-10](https://user-images.githubusercontent.com/32114380/49324714-874d5500-f56e-11e8-97e5-6bc9635b9831.gif)

You can see here that I changed my font to Fura Code NF, and I installed `powerlevel9k`. I expect to see 🐱 when I `echo "🐱"`, but no, I still can't see Emojis.

Also, when I started using `powerlevel9k`, I came across another problem as issued here: [Strange Character / Segment Spacing](https://github.com/bhilburn/powerlevel9k/wiki/Troubleshooting#strange-character--segment-spacing). 

![image](https://user-images.githubusercontent.com/32114380/49324756-63d6da00-f56f-11e8-8f8a-ecb7305e55db.png)

I encountered this issue on Hyper, VSCode integrated terminal and Fluent Terminal. Seems that `xterm.js` doesn't cope well with "Double-Width Characters". I'm not sure if there's something FluentTerm can do to fix this. Nevertheless, thanks again.

**Comment by felixse** [comment]
Yeah, we are somewhat limited to what xterm.js does support, but still way better than any parser/renderer I could have written 😄 

Clickable links should be no big issue, I will put this on my todo. The crash when changing the opacity might be prevented by adding some delay to the update. I didn't encounter it so far on my machines, so I'm just guessing here

**Comment by spencerwooo** [comment]
Okay! Anyway, thank you again for such an amazing, fluent terminal! I seriously consider recommending it over Hyper on my [tutorial repo](https://github.com/spencerwooo/dowww) where I introduce how to make WSL more pretty and work more seamlessly along with VSCode (But it's in Chinese). I look forward to seeing FluentTerm on the Windows Store! 👍👍👍

**Comment by Nufflee** [comment]
I also have the same issue with `clear` and crashing when changing opacity

**Comment by Nufflee** [comment]
ah, thanks @ericcornelissen  :(

**Comment by spencerwooo** [comment]
@Nufflee Even not updated, FluTe is still much better, **faster** and prettier than similar products. (And I'm talking about Hyper, Terminus and other Electron based terminals.)

**Comment by Nufflee** [comment]
@spencerwooo  It's awesome but it keeps crashing on me all the time out of nowhere and that completely beats the purpose..

**Comment by spencerwooo** [comment]
@felixse I think crashing when changing opacity is fixed with v0.3. Thanks ❤️

**Comment by themobiledev** [comment]
I'm on version 0.3.1.0, Windows 10.0.1753 and am seeing the blank terminal. The text appears once I start typing, but if I enter clear in PowerShell or WSL, the prompt and all text disappear again.

As a note, I'm also running this on a 4K monitor but am also seeing it when I move my terminal to my 1080p monitor as well.

**Comment by EvangelosKolimitras** [comment]
Hey guys! I just have an issue with the Terminal only when I use WSL.
For powershell and CMD it behaves, well normal but without any styling since the styling only applies to a WSL Terminal and not to a powershell which I am not fully aware why!

Here is a screenshot

![Desktop _ Shot with GeForce](https://user-images.githubusercontent.com/35934542/69008324-6b8eba80-0949-11ea-84ee-4652b8487147.gif)

Any ideas for a workaround?

Thanks!
